### PR TITLE
styling on homepage, setting up router for posting to cloudinary from create auction

### PR DESCRIPTION
**Pull request by boybutcher**

### CODE REVIEW

File: client/src/components/Home.js [hunk 1]
```diff
@@ -9,54 +10,77 @@ const clickArt = (artId, history) => {
...
+        original: `${item.artwork.image_url}?${item.artwork.id}`,
+        description: `${item.first_name} ${item.last_name} Closing Price: $${item.buyout_price}`, 
+      }
+      images.push(imageObj);
+    })
     return (
-      <Container className="ui small images">
-       {mainArts.map(mainArt => <MainArt key={mainArt.id} art={mainArt} history={history}/>)}
-      </Container>
+      <div style={{'max-height': '400px',
```
Comment (alisonzthu) [inline client/src/components/Home.js]: Is there a way for us to move the css into a css file instead of inline in the code?

Comment (abianco3) [inline client/src/components/Home.js]: I agree. Is there any reason we need inline styles?

Comment (boybutcher) [inline client/src/components/Home.js]: I was intending on putting it into a css file i just wanted to start working on functionality in CreateAuction. If you guys want to wait, I'll plan on moving the css into a file tomorrow (sunday)

Comment (abianco3) [inline client/src/components/Home.js]: Ok Bryan. Let's make sure to add this to Trello as a Refactor/Tech debt Card.

**Comment by boybutcher** [comment]
**READ THIS BEFORE MERGING**
I commented out the 2 lines of using body-parser in our middleware in server.js. I think busboy should work in its place. If any of your server interactions are not working remove busboy and try using body-parser to see if it'll resolve the issue

File: client/src/components/CreateAuction.jsx [hunk 2]
```diff
@@ -23,12 +25,13 @@ class CreateAuction extends React.Component {
...
+    this.handleImageInput = this.handleImageInput.bind(this);
+    this.handleImageCreate = this.handleImageCreate.bind(this);
   }
 
   handleInputChange(e) {
     const name = e.target.name;
     const value = e.target.value;
-    // console.log(`target name: ${name}\ntarget value ${value}`);
     this.setState({
       [name]: value
```
Comment (alisonzthu) [inline client/src/components/CreateAuction.jsx]: is the pair of brackets on line 36 around "name" some new syntax in ES6? I failed to search for its usage

File: client/src/components/CreateAuction.jsx [hunk 3]
```diff
@@ -84,50 +88,163 @@ class CreateAuction extends React.Component {
...
+
+  handleImageCreate() {
+    var formData = new FormData();
+    formData.append('image_file', document.getElementById('imageToSend').files[0]);
+    fetch('/images', {
+        method: 'post',
+        body: formData
+    })
+    .then((data) => {
+      console.log('posting image to Cloudinary SUCCESS! data: ', data);
```
Comment (alisonzthu) [inline client/src/components/CreateAuction.jsx]: we will all need to get rid of console.log()'s for our final code. :)

File: client/src/components/CreateAuction.jsx (hunk 2 above)
Comment (boybutcher) [inline client/src/components/CreateAuction.jsx]: name is a reference to the variable on line 33. I set a name property on each div, so it'll take that divs name, which in this case is the same name as the state key, and itll change that key to the entered value

File: client/src/components/CreateAuction.jsx (hunk 3 above)
Comment (boybutcher) [inline client/src/components/CreateAuction.jsx]: yikes! I'll get rid of the console logs. Sorry!

File: client/src/components/NavBar.jsx [hunk 4]
```diff
@@ -3,51 +3,53 @@ import { NavLink } from 'react-router-dom';
...
+          <NavLink style={linkStyling} to="/artists" >Artists</NavLink>
+          {' | '}
+          <NavLink style={linkStyling} to="/createAuction" >Create an auction</NavLink>
+          {' | '}
+          <NavLink style={linkStyling} to="/events" >Events</NavLink>
+          {' | '}
+          <NavLink style={linkStyling} to="/contactus" >Contact us</NavLink>
           {username === '' ? LoggedOutNav : LoggedInNav}
           {this.props.children}
+          <input style={{'height': '26px', 'font-size': 18, 'margin-left': '20px'}}type="text" placeholder="search" />
```
Comment (alisonzthu) [inline client/src/components/NavBar.jsx]: could the styling in line 52 and line 6 go to a css file, too?

Comment (boybutcher) [inline client/src/components/NavBar.jsx]: ah!! i forgot i had those there..

File: client/src/components/CreateAuction.jsx (hunk 2 above)
Comment (alisonzthu) [inline client/src/components/CreateAuction.jsx]: I see~ Thanks for the explanation!

File: client/src/components/CreateAuction.jsx (hunk 3 above)
Comment (alisonzthu) [inline client/src/components/CreateAuction.jsx]: No sorries. Haha, I got some of mine in the PR as well. Just a reminder for all of us.

File: client/src/components/NavBar.jsx (hunk 4 above)
Comment (alisonzthu) [inline client/src/components/NavBar.jsx]: I'll approve after the styling changes then. Good job!

**Pull request merged**